analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -38,6 +38,7 @@
 # will not be included in the output Clang command.
 IGNORED_OPTIONS = [
     # --- UNKNOWN BY CLANG --- #
+    '-fstack-reuse',
```

Comment:
The flags in this list are in alphabetical order, but apart from that it looks good to me.